src/logging/NewRelicLoggingService.test.js

```diff
@@ -41,17 +41,55 @@ describe('NewRelicLoggingService', () => {
     });
 
     it('calls New Relic client to log message if the client is available', () => {
-      const message = 'Test log';
-      service.logInfo(message);
-      expect(global.newrelic.addPageAction).toHaveBeenCalledWith('INFO', { message });
+      const msg = 'Test log';
+      service.logInfo(msg);
+      expect(global.newrelic.addPageAction).toHaveBeenCalledWith('INFO', { message: msg });
```

Comment:
JS tip, non-blocking: If you changed this variable in order to avoid typing `{ message: message }`, a cool JS trick is that you can just write `{ message }`, which JS will interpret as "pass the variable `message` in as the value for the key `message`".

Reply:
I like this - and I'm changing it.